src/pages/articles/javascript/how-to-create-a-countdown-timer/index.md

```diff
@@ -3,13 +3,47 @@ title: How to Create a Countdown Timer
 ---
 ## How to Create a Countdown Timer
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/articles/javascript/how-to-create-a-countdown-timer/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Methods used:
+* [setInterval()](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval)
+* [clearInterval()](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearInterval)
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+### Creation
+> Start by building the countdownTimer function.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+```javascript
+function startCountdown(seconds){
+  var counter = seconds;
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+  var interval = setInterval(() => {
+    console.log(counter);
+		counter--;
+		if(counter < 0 ){
+		  clearInterval(interval);
+			console.log('Ding!');
+		};	
+  }, 1000);
+};
+```
 
+### Execution
+> Now to start the timer we provide `startCountdown()` with a number value as an argument which represents the seconds to countdown.
 
+```javascript
+  startCountdown(10)l
+  // Console Output // 
+  // 10
+  // 9
+  // 8
+  // 7
+  // 6
+  // 5
+  // 4
+  // 3
+  // 2
+  // 1
+  // 0 
+  // Ding!
+```
+
+### Live Example
+[Codepen - Countdown Timer](https://codepen.io/rdev-rocks/pen/jLogxY?editors=1111)
```

Comment:
Can you change this link to https://codepen.io/rdev-rocks/pen/jLogxY?editors=0012 so that only the required panels will show?

Reply:
I think I understand, I changed the view to be more focused with the panels and console. ty